src/main/java/org/jboss/xavier/analytics/pojo/input/VMWorkloadInventoryModel.java

```diff
@@ -0,0 +1,179 @@
+package org.jboss.xavier.analytics.pojo.input;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Map;
+
+
+public class VMWorkloadInventoryModel
+{
+    //common/name
+    private String provider;
+    //common/ems_clusters/v_parent_datacenter
+    private String datacenter;
```

Comment:
Which one of the ems_clusters would be referred ?

Reply:
the spreadsheet describes how to find the correct ems_cluster